### PR TITLE
Qualcomm AI Engine Direct - fix GA models breakage after HF uplevel

### PR DESCRIPTION
**Pull request by haowhsu-quic**
### Summary
- bert series: pre-calculate attention mask & adjust observer eps
- mobilevit_v1 / efficientnet / whisper: adjust observer eps
- t5: adopt HF 5.0 codebase changes
- llama: resolve HF 5.0 conflicts
- fix per-tensor dump naming collision in node visitor

### Test plan
Please refer to `backends/qualcomm/tests/test_qnn_delegate.py`

### CODE REVIEW

**Comment by haowhsu-quic** [comment]
Hi @cccclai, this PR helps fix GA model breakages after transformer update. Please have a look, thank you.

**Review by cccclai — APPROVED** [review]

**Comment by mergennachin** [comment]
This is past the 1.1 branch cut, does it need to be cherry-picked? If so, please look at cherry-pick instructions at https://github.com/pytorch/executorch/issues/16365

**Comment by mergennachin** [comment]
cc @haowhsu-quic
